packages/@webex/widget-cc-station-login/src/station-login/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {useStationLogin} from './station-login.hooks';
+import StationLoginPresentational from './station-login.presentational';
+
+export const StationLogin: React.FunctionComponent = () => {
```

Comment:
Can we use a named function instead of an arrow function. Named function helps with debugging. This applies to every components we have created.